docs/wdc_debugging.md

```diff
@@ -4,9 +4,8 @@ layout: docs
 ---
 
 You can use the simulator included with the WDC SDK to develop and debug your connectors more quickly. The simulator
-is a JavaScript application that loads connectors in a similar way to Tableau Desktop. Additionally, Tableau
-Desktop includes a built-in (although not supported) debugger that you can use to debug issues that appear in Tableau Desktop
-but not in the simulator.
+is a JavaScript application that loads connectors in a similar way to Tableau Desktop. Additionally, starting with Tableau Desktop 2019.4, you can use the Chrome DevTools to debug issues that appear in Tableau Desktop, but not in the simulator. 
+In versions of Tableau Desktop prior to 2019.4, you can use a built-in--although, not supported--debugger that you can use to debug WDC issues that appear in those versions of Tableau Desktop.
```

Comment:
Is this how it's supposed to look?
"built-in--although, not supported--debugger"

Reply:
Well, I didn't have them quite right. They are supposed to be "em" dashes, which are used to set off parenthetical but important information in a sentence.  Turns out that it needs to be `---` dashes.  They work correctly in my local Jekyll build. They should be fine on GitHub. 
Like this: 
built-in—although, not supported—debugger